src/Symfony/Component/Messenger/Worker.php

```diff
@@ -290,7 +290,7 @@ public function stop(): void
         $this->shouldStop = true;
     }
 
-    public function keepalive(): void
+    public function keepalive(?int $seconds): void
```

Comment:
As this is a public method (API) and it has no default value, this is a BC break, isn't it?